packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Statements/SwitchCaseStatement.cs

```diff
@@ -43,8 +43,10 @@ internal override void Write(CodeWriter writer)
                 writer.AppendRaw("default");
             }
 
-            using var scope = writer.ScopeRaw(":", "", false);
-            Statement.Write(writer);
+            using (writer.ScopeRaw(":", string.Empty, false))
```

Comment:
nit: why did we change to using a block scope here?

Reply:
Is it necessary to pass false here? Since we are not at the beginning of the line I expected that we wouldn't add the indentation anyway. I ask because I'm wondering if we can do away with the addSpaces param and instead just not add spaces when the line starts with a `#`.

Reply:
hey @JoshLove-msft, thanks for pointing this out. You're right, the block isn't needed. I went ahead and reverted this back. Originally, I had made some other [changes](https://github.com/microsoft/typespec/pull/3603/files#diff-5a5e5c753fc475ae65640aacc837c2ee5c00f98684e51519a2884a1d8dda6d2e) to this statement and I pulled a snippet of that over. 

Regarding passing in `false`, that param actually refers to the `newLine` param. I went ahead and added it as a named parameter for clarity. If it's not added, then an additional new line is added at the end of each switch case scope. Here is that change outlined in the unit test I added:
![image](https://github.com/microsoft/typespec/assets/102122018/d737ce6a-0ad5-4435-9dd3-208785184e5a)

I also went ahead and took your tip of just checking if the span character is not equal to `#`, rather than an additional param to these APIs. Thank you.